### PR TITLE
YAML-based issue forms

### PR DESCRIPTION
**Pull request by pivotman319-new**
This pull request removes the legacy Markdown issue templates and replaces them with the newer YAML-based issue forms.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      description: Enter a description of what actually happened when reproducing this issue.
+      placeholder: The program crashed after parameter Z was being set in X.
+  - type: textarea
+    attributes:
+      label: Screenshots/Log Output
+      description: If applicable, add screenshots or log output here to help further explain your problem.
+  - type: textarea
+    attributes:
+      label: Additional details
+      description: Please provide any additional details that you would like to share here. You can leave this field blank.
```
Comment (nibanks) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:64]: Let's combine these and just have an Additional details section.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      placeholder: A crash occurs if X is called from Y.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Operating system version
+      description: On what operating system did this bug occur? Provide information of the affected OS and its version here.
+      placeholder: |
+        Windows 11 (Build 10.0.22463.1000.rs_prerelease.210917-1503), Ubuntu 20.04 LTS
+  - type: textarea
```
Comment (nibanks) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:21]: What do you think about having a drop down that contains all the (pre)release branches and `main` (default to main)?

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+labels: []
+body:
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the bug
+      description: Please enter a clear and short description of your bug here.
+      placeholder: A crash occurs if X is called from Y.
+  - type: textarea
```
Comment (nibanks) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:13]: A thought: a set of checkboxes for the affected OSes, and then possibly a separate section for additional details?

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: ⭐ Submit a feature request
+description: Request a feature that you think should be supported
+title: "Feature request title"
+labels: []
```
Comment (nibanks) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:4]: Please add `feature request` to the list.

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 2 above)
Comment (pivotman319-new) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:21]: Sounds good, I'll work on that.

**Review by nibanks — APPROVED** [review]

**Pull request merged**